modules/updating-eus-to-eus-upgrade.adoc

```diff
@@ -18,6 +18,7 @@ Following this procedure reduces the total update duration and the number of tim
 If your cluster is running a version than {product-title} 4.8.14, you must update to a later 4.8.z version prior to updating to 4.9.
 The update to 4.8.14 or later is necessary to fulfill the minimum version requirements that must be performed without pausing MachineConfigPools.
 * Verify that MachineConfigPools is unpaused.
+* Update OpenShift CLI`oc` to version 4.9.
```

Comment:
Quite agree with https://bugzilla.redhat.com/show_bug.cgi?id=2067291#c2. I just had a little concern about the oc's version because i'm not sure if we could always keep 4.9 consistent with 4.10. Take https://bugzilla.redhat.com/show_bug.cgi?id=2065507 for example, we plan to backport it to 4.10 but not to 4.9 together with https://bugzilla.redhat.com/show_bug.cgi?id=2064991. At that time, the oc's behavior should be different in v4.9 and v4.10.
So, i suggest we address it in this pr to be `Update OpenShift CLI oc to the version for your updated version before each upgrade`. 
Of course, if we have another pr to track this for a later revert, i'm ok to get this pr merged asap.

Reply:
That sounds good, I'll update the PR.

Reply:
I added a note about requiring 4.9 for the `oc adm upgrade channel` command too.